### PR TITLE
DP-21655 Upgrade to PHP 7.4

### PR DESCRIPTION
**Pull request by weitzman**
Also updates to PHPUnit 7 and Drush 10.5.0

**Jira:** (Skip unless you are MA staff)
https://jira.mass.gov/browse/DP-21655

**To Test:**
- [ ] If its green its good.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -2,7 +2,8 @@ version: '3.1'
 
 services:
   drupal:
-    image: "massgov/drupal-container:1.0.17-dev"
+    #image: "massgov/drupal-container:php74-beta2-dev"
+    build: /Users/moshe.weitzman/reps/Drupal-Container/dev
```
Comment (oknate) [inline docker-compose.yml]: Is this right, it looks like a local mac path?

Comment (oknate) [reply docker-compose.yml]: should it be `image: "massgov/drupal-container:php74-beta2-dev"`?

**Review by oknate — CHANGES_REQUESTED** [review]
docker-compose.yml needs update, see comment.

File: docker-compose.yml (hunk 1 above)
Comment (weitzman) [reply docker-compose.yml]: Good catch. Its now `massgov/drupal-container:1.0.19-dev` which contains today's bugfix for drupal-container. I will fix composer conflicts next.

**Comment by weitzman** [comment]
I can confirm that https://github.com/massgov/Drupal-Container/pull/21 did its job and Xdebug defaults to Off again.

**Review by oknate — APPROVED** [review]
looks good now, local runs php 7.4; xdebug fix works, it's disabled by default now.

**Pull request merged**